tests/test_fyle/test_helpers.py

```
@@ -440,3 +444,27 @@ def test_get_source_account_type():
+def test_assert_valid_request():
+    """Test assert_valid_request with caching"""
+    workspace = Workspace.objects.create(
+        name='Test Workspace',
+        fyle_org_id='test_org_123'
+    )
+    assert_valid_request(workspace.id, 'test_org_123')
+    assert_valid_request(workspace.id, 'test_org_123')
+
+    try:
+        assert_valid_request(999, 'test_org_123')
+        assert False, "Should have raised ValidationError"
```

Ruff (0.14.1)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)